custom_components/tahoma/const.py

```diff
@@ -9,28 +9,29 @@
 from homeassistant.components.switch import DOMAIN as SWITCH
 from homeassistant.components.water_heater import DOMAIN as WATER_HEATER
 
-CONF_HUB = "hub"
 DOMAIN = "tahoma"
 
-DEFAULT_HUB = "Somfy TaHoma"
+CONF_HUB = "hub"
+DEFAULT_HUB = "Somfy (Europe)"
```

Comment:
This will break behavior for current users. I still need to add a fallback in case the hub can't be found. Don't want to add a migration to this custom integration.

Reply:
@tetienne @vlebourl what do you think would be the best?

I would love to rebrand this repo + the integration to overkiz. Perhaps we can do this at the same moment? It will be a breaking change, but at least gives us the opportunity to do two breaking changes at the same time ;-).